udunits2/udunits2.spec

```
@@ -0,0 +1,243 @@
+Name: udunits2
+Version: 2.2.28
+Release: 7%{?dist}
+Summary: A library for manipulating units of physical quantities
+License: UCAR
+URL: http://www.unidata.ucar.edu/software/udunits/
+Source0: ftp://ftp.unidata.ucar.edu/pub/udunits/udunits-%{version}.tar.gz
+# default_udunits2_xml_path() is buggy and broken
+Patch0: udunits-2.2.28-fix-xml-path-logic.patch
+BuildRequires: make
+BuildRequires: gcc-c++, groff, byacc, expat-devel, CUnit-devel
+BuildRequires: chrpath
+BuildRequires: /usr/bin/makeinfo
+BuildRequires: /usr/bin/texi2dvi
```

Why do we require an absolute path here ?

```
dnf provides texi2dvi
Last metadata expiration check: 0:30:04 ago on Thu 08 Jun 2023 07:06:05 UTC.
texinfo-tex-6.8-4.oe2203sp1.aarch64 : Tools for formatting Texinfo documents
Repo        : @System
Matched from:
Filename    : /usr/bin/texi2dvi
```

Since this .spec file is specifically for openEuler I think we can do this:
```suggestion
BuildRequires: texinfo-tex
```